lib/fastlane/plugin/wpmreleasetoolkit/actions/common/check_translation_progress.rb

```diff
@@ -0,0 +1,135 @@
+module Fastlane
+  module Actions
+    class CheckTranslationProgressAction < Action
+      def self.run(params)
+        require_relative '../../helper/glotpress_helper.rb'
+
+        UI.message('Check translations status...')
```

Comment:
Nitpick, what do you think of "checking" to explain the action is happening right now? "check" makes me think the tool is telling the user to go and check.

 ```suggestion
        UI.message('Checking translations status...')
```